### ISSUE TITLE
Continue to play Youtube video on mute in background

### ISSUE THREAD

**Issue by tobyurff** [cpiber/NebulaEnhance#44]
Youtube ranks and recommends videos, and pays creators, based on videos that were played (meaning at least 30 seconds have been watched). When we use the extension, currently the video will stop playing after 3 seconds and we continue to watch on Nebula.

I think a great enhancement would be to not stop the video, but instead mute it and let it continue play in the background, so creators get credit on Youtube for our interest.

**Comment by cpiber** [comment]
That is probably even easier than stopping it, since YouTube really does not want you to stop a video ^^ I'll look into that

**Comment by cpiber** [comment]
I have implemented this now, please download this version to try it out: 
[enhancer_for_nebula-1.3.1.1.zip](https://github.com/cpiber/NebulaEnhance/files/11242639/enhancer_for_nebula-1.3.1.1.zip)

To install:
- in Chrome, please go to Setting, click Extensions (lower left), enable Developer Mode (upper right) and select "Load unpacked". Here select the unzipped folder (should contain folders _locale, icons, scripts, styles).
- in Firefox, please go to about:addons, click on the gear (top right), select Debug addons and click "Load Temporary Addon". Here select the zip you downloaded.

**Comment by tobyurff** [comment]
@cpiber Awesome, I've tried it out with a few videos and it seems to work nicely.

The only small downside I found is that when using Youtube after watching the Nebula version of the video, it retains the information that videos play muted. E.g. if I open Youtube's homepage and click on any video, I'll need to unmute it to watch with sound again. I wouldn't know how to "fix" that though unless there's a way to easily hook into the video finishing on Youtube to undo the muting.
